### ISSUE TITLE
Null Pointer Exception when using v1.4.0 with ES 0.90.x

### ISSUE THREAD

**Issue by MattDevUK** [s12v/newrelic-elasticsearch#7]
When installing plugin version 1.4.0 using the NPI alongside ElasticSearch version 0.90.x, it returns:

```
java.lang.NullPointerException
        at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportCalculatedClusterStats(Unknown Source)
        at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportNodesStats(Unknown Source)
        at me.snov.newrelic.elasticsearch.ElasticsearchAgent.pollCycle(Unknown Source)
        at com.newrelic.metrics.publish.Runner$PollAgentsRunnable.run(Runner.java:209)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
SEVERE: An error has occurred
```

ElasticSearch version: 0.90.13
JVM Version: 1.7.0_75

**Comment by s12v** [comment]
Probably some stats were not available in 0.9... I'll try to reproduce it, thanks for the report.

**Comment by s12v** [comment]
@MattDevUK, please upgrade the plugin

**Comment by s12v** [comment]
please note that many of the stats are not available in 0.90, so many graphs will be empty
